src/App.tsx

```diff
@@ -17,35 +19,39 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Home
+          </NavLink>
+          <NavLink
+            to="/tabs"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Tabs
+          </NavLink>
         </div>
       </div>
     </nav>
 
     <div className="section">
       <div className="container">
-        <h1 className="title">Home page</h1>
-        <h1 className="title">Tabs page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="tabs is-boxed">
-          <ul>
-            <li data-cy="Tab" className="is-active">
-              <a href="#/">Tab 1</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 2</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="TabContent">
-          Please select a tab
-        </div>
+        <Routes>
+          <Route path="/" element={<h1 className="title">Home page</h1>} />
```

Comment:
if you don't want to create a separate Homepage page, maybe it's better to write everything here on a new line, because it's hard to read